src/views/Home.js

```diff
@@ -23,8 +23,8 @@ export function DefaultPage({ children }) {
   });
 
   return (
-    <div className="h-screen w-100">
-      <div className="h-full w-full absolute z-10 flex items-center">
+    <div className=" w-full">
```

Comment:
@aliraza944 can we retain the height and width specifications, i.e. "h-screen w-100" in order to be able to display the map? With just "w-full" and no inherited width property to define it, nothing is set and the map does not show.